js/backend.js

```diff
@@ -35,10 +32,9 @@ $('#trainNew').on('click', function () {
             // on close get file path
             if (data.filePaths[0]) {
                 console.log("Changing paths to: '" + data.filePaths[0] + "'");
-                modelPath = path.join(data.filePaths[0].goodPath(), modelName);
```

Comment:
yeah you right.  I added this thinking 'load new' was specifying a dir to house a new model, but if this is for 'load existing' then we're golden, and we can just add a 'create blank model' button using the saveDialog instead. In either case we don't need the name